omod/src/main/java/org/openmrs/module/patientqueuapp/page/controller/DeliveryRoomQueuePageController.java

```diff
@@ -0,0 +1,34 @@
+package org.openmrs.module.patientqueuapp.page.controller;
+
+import org.openmrs.Concept;
+import org.openmrs.module.kenyaui.annotation.AppPage;
+import org.openmrs.ui.framework.page.PageModel;
+import org.openmrs.api.context.Context;
+import org.openmrs.ui.framework.UiUtils;
+import org.openmrs.ui.framework.page.PageRequest;
+
+
+import javax.servlet.http.HttpSession;
+import java.util.Date;
+
+/**
+ * Created by George Bush 15/06/2020
+ */
+public class DeliveryRoomQueuePageController {
+    private static final String DELIVERY_ROOM_CONCEPT_UUID = "to be generated";
```

Comment:
What concept is corruntly used? can we have an equivalent in KenyaEMR or if missing we create one?